bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/Channel.java

```diff
@@ -77,19 +82,35 @@ public Channel(ChannelUID uid, String acceptedItemType) {
         this.properties = Collections.unmodifiableMap(new HashMap<String, String>(0));
     }
 
+    /**
+     * @deprecated - use {@link ChannelBuilder} instead
+     */
+    @Deprecated
     public Channel(ChannelUID uid, String acceptedItemType, Configuration configuration) {
         this(uid, null, acceptedItemType, ChannelKind.STATE, configuration, new HashSet<String>(0), null, null, null);
     }
 
+    /**
+     * @deprecated - use {@link ChannelBuilder} instead
+     */
+    @Deprecated
     public Channel(ChannelUID uid, String acceptedItemType, Set<String> defaultTags) {
         this(uid, null, acceptedItemType, ChannelKind.STATE, null, defaultTags, null, null, null);
     }
 
+    /**
+     * @deprecated - use {@link ChannelBuilder} instead
+     */
+    @Deprecated
     public Channel(ChannelUID uid, String acceptedItemType, Configuration configuration, Set<String> defaultTags,
             Map<String, String> properties) {
         this(uid, null, acceptedItemType, ChannelKind.STATE, null, defaultTags, properties, null, null);
     }
 
+    /**
+     * @deprecated - use ChannelBuilder instead
+     */
+    @Deprecated
```

Comment:
Hm, so the `ChannelBuilder` itself is using a `Deprecated` function...

Reply:
Yes, and I don't intend to take it away from _him_.

In an ideal world, we would convert the `Channel` class to an interface and hide the implementation class in a private package, allowing the channel builder to access its then non-deprecated constructors (I'm pretty sure that's what you would have done too). But then a couple of people might set a bounty on my head when they discover the `IncompatibleClassChangeError`s  😉 

I'm really looking forward towards our yet outstanding, huge, big, breaking, disastrous API cleanup initiative that we will have to do before a potential 1.0 release (but hardly dare really doing it).

Reply:
> I'm pretty sure that's what you would have done too

:+1: :wink: